cdap-hbase-compat-1.1/src/main/java/co/cask/cdap/data2/transaction/messaging/coprocessor/hbase11/MessageTableRegionObserver.java

```diff
@@ -180,6 +186,44 @@ public void postCompact(ObserverContext<RegionCoprocessorEnvironment> e, Store s
     }
   }
 
+  @Override
+  public void prePut(ObserverContext<RegionCoprocessorEnvironment> e, Put put, WALEdit edit,
```

Comment:
I prefer not to rely on coprocessor in TMS for this. Can we do it in client side?